strings.js

```diff
@@ -0,0 +1,37 @@
+// 1. Uppercase the first character
+
+function ucFirst(str) {
+    return str[0].toUpperCase() + str.slice(1).toLowerCase();
+}
+
+let text = 'teXt';
+console.log(ucFirst(text));
+
+// 2. Check for spam
+
+function checkSpam(str) {
+    const LowerCaseStr = str.toLowerCase();
```

Comment:
use variable names that start with upper case only for classes and constructors

Reply:
done